packages/webapp/src/utils/appinsights.ts

```diff
@@ -28,6 +28,10 @@ export const appInsights = new ApplicationInsights({
 })
 appInsights.loadAppInsights()
 
+export function setTelemetryTracking(isEnabled: boolean) {
+	appInsights.config.disableTelemetry = isEnabled
```

Comment:
there seems to be a mismatch between the parameter name (isEnabled) and the config name (disableTelemetry). I think it would make more sense to name the parameter isDisabled to have both coordinated. Unless I misunderstood what is going on of course.

Reply:
Ya, I see what you mean